demo/components/date-picker-demo.tsx

```diff
@@ -31,3 +31,52 @@ export class DatePickerDemo extends React.Component<Partial<DatePickerProps>, Da
         this.setState({value: updatedDate });
     }
 }
+
+export class DatePickerDemoStartingDay extends React.Component<Partial<DatePickerProps>, Partial<DatePickerDemoState>> {
+    public componentWillMount() {
+        this.setState({
+            value: this.props.value ? this.props.value : new Date(),
+            startingDay: 0
+        });
+    }
+
+    public render() {
+        return (
+            <div>
+                <h2>DatePicker with variable week (and showDropdown set to true)</h2>
```

Comment:
Please change the phrasing to be more understandable. Since this is a demo you can say "DatePicker with Wednesday as the first day of the week!"